content/applications/finance/accounting/taxation/taxes/taxcloud.rst

```diff
@@ -1,130 +1,126 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 * For each one of your companies that uses TaxCloud, it might be necessary to create a 
   **User-Defined Default** so that the new taxes generated by the TaxCloud integration are created 
-  with the correct Tax Payable account:
+  with the correct **Tax Payable** account:
   
 .. warning::
    A User-Defined Default impacts all records at creation.  It means that **every** new tax will be 
    set up to record income in the specified Tax Payable account, unless the tax is manually edited
    to specify a different income account (or there exists another User-Defined Default that takes 
    precedence).
 
-* In :menuselection:`Accounting --> Configuration --> Chart of Accounts`, select the Tax Payable
-  account for the company. Take note of the account's ``id`` in the URL string. 
+* In :menuselection:`Accounting Dashboard --> Configuration --> Chart of Accounts`, select the Tax
+  Payable account for the company. Take note of the account's ``id`` in the URL string.
 
 .. image:: taxcloud/user-default-find-account-id.png
-   :alt: The account's ID can be found in the URL string as 'id=...'. 
    :align: center
+   :alt: The account's ID can be found in the URL string as 'id=...'.
 
 * Activate the :ref:`developer mode <developer-mode>`, then go to 
-  :menuselection:`Settings --> Technical --> Actions --> User-Defined Defaults`, and 
-  click on *Create*.
+  :menuselection:`Settings App --> Technical --> Actions: User-Defined Defaults`, and click on
+  :guilabel:`Create`.
 
-* Click on *Field*, then, in the drop-down menu, on *Search More*. 
+* Click on :guilabel:`Field`, then, in the drop-down menu, on :guilabel:`Search More`.
 
 .. image:: taxcloud/user-default-search-field.png
    :alt: Click on 'Search More' in the 'Field' drop-down menu. 
    :align: center
 
-* In the pop-up's search box, filter on the model ``tax.repartition.line`` and the field ``account``. 
-  Select the ``account`` field of the ``tax.repartition.line`` model. 
+* In the pop-up search box, filter on the *model* `tax.repartition.line` and the *field* `account`.
+  Select the line `tax.repartition.line`.
 
 .. image:: taxcloud/user-default-select-field.png
    :alt: Select the 'account' field of the 'tax.repartition.line' model. 
    :align: center
 
-* In the **Default Value** field, enter the ID of the company's Tax Payable account. 
-  Select the company for which this configuration should apply in the *Company* field. 
-  Click *Save*. 
+* In the **Default Value (JSON format)** field, enter the ID of the company's Tax Payable account.
+  Select the company for which this configuration should apply in the :guilabel:`Company` field.
+  Click :guilabel:`Save`.
 
 .. image:: taxcloud/user-default-enter-default-account-id.png
-   :alt: Enter the ID of the company's Tax Payable account. 
    :align: center
-
+   :alt: Enter the ID of the company's Tax Payable account.
 
 How it works
```

Comment:
shouldn't we avoid title like "how to" ?

Reply:
I think "How to" titles are generally used to instruct the reader on specific steps to achieve a certain configuration, etc. In this case, it's the general functioning of TaxCloud, not "how to activate X feature". Imo, it can remain as is.